Wikipedia/Code/WMFReferencePageViewController.swift

```diff
@@ -0,0 +1,108 @@
+
+import Foundation
+
+extension UIViewController {
+    class func wmf_viewControllerFromReferencePanelsStoryboard() -> Self {
```

Comment:
Shouldn't this just be a category on this view controller, not all view controllers?